core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -1156,6 +1158,433 @@ describe('Contributions and review component', () => {
         });
       });
 
+      it('should load reviews, then contributions', fakeAsync(() => {
+        getReviewableTranslationSuggestionsAsyncSpy.and.returnValue(
+          Promise.resolve({
+            suggestionIdToDetails: {
```

Comment:
Can we assign this object to a variable so that we can reference it in the `toHaveBeenCalledWith` calls below?

Reply:
Done